client/src/components/button/DropBoxButton.jsx

```
@@ -1,11 +1,31 @@
-    </div>
-  )
+    <>
+      {openModal && (
+        <AlertModal clickHandler1={closeModalHandler} clickHandler2={clickHandler2}>
+          리뷰를 삭제하시겠습니까?
+        </AlertModal>
+      )}
+      <div className={styles.wrapper}>
+        <div className={styles.btn} onClick={clickHandler1}>
+          수정
+        </div>
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)